simp/compliance_profiles/testdata.yaml

```diff
@@ -0,0 +1,20 @@
+---
+version: 2.0.0
+profiles:
+  disa:
+    checks:
+      disacheck: true
+  nist:
+    checks:
+      nistcheck: true
+checks:
+  nistcheck:
+    type: 'puppet-class-parameter'
```

Comment:
Shouldn't this just be the default and/or discovered from the environment? I mean, we can inspect and see if we're running inside Puppet.

Reply:
no, every check is explicitly typed